### PR TITLE
static: remove perl dependency

### PR DESCRIPTION
**Pull request by kubiko**
remove perl dependency in base snap

Signed-off-by: Ondrej Kubik <ondrej.kubik@canonical.com>

Thanks for helping us make a better ubuntu core!

Before continuing with the PR, you might want to consider also creating a PR for the new core-base repository
at https://github.com/snapcore/core-base, which contains newer core versions (22+), if the PR is also relevant
for newer core versions.

### CODE REVIEW

**Review by valentindavid — APPROVED** [review]
Approved.

**Comment by valentindavid** [comment]
core-base PR snapcore/core-base#21

File: static/usr/bin/core-sshd-host-keygen [hunk 1]
```diff
@@ -9,10 +9,8 @@ get_config_option() {
 	[ -f /etc/ssh/sshd_config ] || return
 
 	# TODO: actually only one '=' allowed after option
-	perl -lne '
-		s/[[:space:]]+/ /g; s/[[:space:]]+$//;
-		print if s/^[[:space:]]*'"$option"'[[:space:]=]+//i' \
-	   /etc/ssh/sshd_config
+	awk -v option=$option '$1 == option {print $2}' \
+        /etc/ssh/sshd_config
```
Comment (alfonsosanchezbeato) [inline static/usr/bin/core-sshd-host-keygen:13]: nitpick: you are mixing tabs (first line) with spaces (second line)

**Review by alfonsosanchezbeato — COMMENTED** [review]
Thanks, looks good! Just a minor nitpick.

**Review by alfonsosanchezbeato — APPROVED** [review]

**Review by anonymouse64 — COMMENTED** [review]
+0 from me because I don't understand perl, but the new awk bit looks correct so I think it's doing the right thing but because I don't know exactly what the original perl was doing I can't say for sure it's correct :smile:

**Comment by kubiko** [comment]
One more change to address also comment from @valentindavid that sshd config is case insensitive.

**Comment by kubiko** [comment]
@anonymouse64 that makes us two! :)
I just checked from the code what it actually expect to get back, rather than understand perl.

**Pull request merged**